### DISCUSSION TITLE
Vertical Thirds

### DISCUSSION THREAD

**Discussion by bam95899** [rxhanson/Rectangle#411] [Ideas]
Allow the ability to split windows vertically by thirds, would be useful on vertical monitors.

**Comment by rxhanson** [comment]
The thirds commands automatically adjust to vertical displays. If this isn’t the case for you, feel free to open a bug.

**Reply by softveda** [reply]
Lucky I read this. It works but without reading the comment I wouldn't have known or tried.
I use a 27" monitor in portrait mode and this essential for me.
May be you can change the help text to "First Third (V/H)"

**Reply by rxhanson** [reply]
There will be some changes to the UI in an upcoming release, where my plan is to add tooltip style information which should solve this issue in an elegant way.
